source/Calamari.CloudAccounts/AwsEnvironmentGeneration.cs

```diff
@@ -46,11 +46,35 @@ public static async Task<AwsEnvironmentGeneration> Create(ILog log, IVariables v
         async Task Initialise()
         {
             PopulateCommonSettings();
-            await PopulateSuppliedKeys();
-            await PopulateKeysFromInstanceRole();
+
+            if (!await PopulateSuppliedKeys())
+            {
+                if (!await LoginFallback())
+                {
+                    throw new Exception("AWS-LOGIN-ERROR-0006: "
+                                        + "Failed to login via external credentials assigned to the worker. "
+                                        + $"For more information visit {log.FormatLink("https://g.octopushq.com/AwsCloudFormationDeploy#aws-login-error-0006")}");
```

Comment:
I think we need to be doing any these links using `oc.to` rather than `g.octopushq.com`

Reply:
Good catch - I'll make those changes.